Products/ZenModel/migrate/addEmergencyShutdownLevel.py

```diff
@@ -0,0 +1,55 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2016, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+import logging
+log = logging.getLogger("zen.migrate")
+
+import Migrate
+import servicemigration as sm
+sm.require("1.1.5")
+
+from levels import levels
+
+
+class AddEmergencyShutdownLevel(Migrate.Step):
+    """
+    Add the emergency shutdown and startup levels to service definitions.
+    See ZEN-23931.
+    """
+
+    version = Migrate.Version(5,2,0)
```

Comment:
We've switched to a different numbering scheme for migrations.  This should now be either 108.0.0 or 109.0.0.